packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx

```diff
@@ -456,7 +458,10 @@ export const CommentTree = ({
 
         return (
           <React.Fragment key={comment.id + '' + comment.markedAsSpamAt}>
-            <div className={`Comment comment-${comment.id}`}>
+            <div
+              className={`Comment comment-${comment.id}`}
+              ref={index === commentsList.length - 1 ? commentsRef : null}
```

Comment:
this would scroll to the last comment, ex: if there are 100 comments, it will scroll all the way down. Lets update it to scroll the first comment into view.